specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2018-07-10/service.json

```diff
@@ -14235,6 +14243,55 @@
       },
       "x-ms-discriminator-value": "A2A"
     },
+    "RecoveryPlanA2ADetails": {
+      "description": "Recovery plan A2A specific details.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/RecoveryPlanProviderSpecificDetails"
+        }
+      ],
+      "properties": {
+        "primaryZone": {
+          "description": "The primary zone.",
+          "type": "string"
+        },
+        "recoveryZone": {
+          "description": "The recovery zone.",
+          "type": "string"
+        }
+      },
+      "x-ms-discriminator-value": "RecoveryPlanA2ADetails"
+    },
+    "RecoveryPlanA2AInput": {
+      "description": "Recovery plan A2A input.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/RecoveryPlanProviderSpecificInput"
+        }
+      ],
+      "properties": {
+        "primaryZone": {
+          "description": "The primary zone.",
+          "type": "string"
+        },
+        "recoveryZone": {
+          "description": "The recovery zone.",
+          "type": "string"
+        },
+        "canWrite": {
```

Comment:
RecoveryPlanProviderSpecificInput brings in canRead\Write, no need to duplicate the definition